README.rst

```diff
@@ -66,6 +66,10 @@ which is roughly:
 Recent Changes
 ==============
 
+- 1.7.0
+
+  - **MSRV**: Rust 1.49 or later is now required.
```

Comment:
We should mention the updated hashbrown :)

Reply:
Done!